rdb/scripts/informix/io_read_count.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/bash
```

Comment:
The Sha-Bang for bash is usually "/bin/bash". "/usr/bin/bash" is not workable for many platforms.